app/tests/test_generic_api.py

```diff
@@ -452,6 +454,11 @@ def test_get_cache_header(self):
                                        timedelta(seconds=3600).total_seconds(),
                                        delta=2)
 
+    def test_get_asset_object_cache_header(self):
+        key = get_asset_path(self.item.model, self.asset["name"])
+        self.assertS3ObjectExists(key)
+        self.assertS3ObjectCacheControl(key, max_age=7200)
```

Comment:
Not sure I understand, why max_age is 7200 here..

Reply:
Default cache settings from the settings.